wgsl/index.bs

```diff
@@ -14132,7 +14327,7 @@ Note: The result is not mathematically meaningful when `abs(t)` &ge; 1.
     <td style="width:10%">Overload
     <td class="nowrap">
       <xmp highlight=wgsl>
-        @const @must_use fn cross(e1: vec3<T>,
+        @const @must_use fn cross(a: vec3<T>,
```

Comment:
You only changed one parameter name and the new description refers to a undefined `b`.